src/session.ts

```diff
@@ -99,7 +99,7 @@ async function updateSession(
     return NextResponse?.redirect
       ? NextResponse.redirect(redirectTo)
       : new Response(null, {
-          status: 302,
+          status: 307,
```

Comment:
should this be a 307? if the original request were POSTing to some endpoint, i think a 307 would have you POST (the original request body) to the authorization url (but we want to GET).